pkg/metrics/internal_user_log.proto

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Is it possible to depend on some other package that includes this file instead of having the proto copied here too?

Reply:
Sure , I can Isolate them in a seperate package.
b/309517469 is created to cover this part